lib/galaxy/jobs/runners/local.py

```diff
@@ -105,7 +105,7 @@ def queue_job(self, job_wrapper):
                                     env=self._environ,
                                     preexec_fn=os.setpgrp)
 
-            proc.terminated_by_shutdown = False
+            proc.terminated_by_shutdown = False  # type: ignore
```

Comment:
now that we have `show_error_codes = True` under `[mypy]` in `setup.cfg`, you can refine these `# type: ignore` to include the specific error code being ignored. Example: `# type: ignore[attr-defined]`